dataproviders/pom.xml

```diff
@@ -0,0 +1,133 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.paypal.selion</groupId>
+        <artifactId>SeLion-Parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>SeLion-dataproviders</artifactId>
+    <packaging>jar</packaging>
+    <name>SeLion DataProviders</name>
+    <description>A collection of SeLion TestNG DataProviders for automation with TestNG.</description>
+
+    <properties>
+        <suiteXmlFile>src/test/resources/suites/Default-Suite.xml</suiteXmlFile>
+    </properties>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>selionbuildinfo.properties</include>
+                </includes>
+            </resource>
+            <resource>
+                <!-- make our test reources available to SeLion client -->
+                <directory>src/test/resources</directory>
+                <filtering>false</filtering>
+                <includes>
+                    <include>**/*.json</include>
+                    <include>**/*.yaml</include>
+                    <include>**/*.xlsx</include>
+                </includes>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
```

Comment:
Make this `dataproviderSuiteXmlFile`... if you build from the parent directory with -DsuiteXmlFile=PhantomJS-Suite.xml this module will not build because there is no such suite.. This is why travis ci has failed your PR -- https://travis-ci.org/paypal/SeLion/builds/71008236